src/components/NewEnvironment/index.js

```diff
@@ -0,0 +1,260 @@
+import React, {useState} from 'react';
+import { Mutation } from 'react-apollo';
+import Button from 'components/Button';
+import Modal from 'components/Modal';
+import gql from 'graphql-tag';
+import withLogic from 'components/NewEnvironment/logic';
+import Image from "next/image";
+import show from "../../static/images/show.svg";
+import hide from "../../static/images/hide.svg";
+import { CopyToClipboard } from "react-copy-to-clipboard";
+import {StyledNewEnvironment} from './StyledNewEnvironment';
+import {useQuery} from "@apollo/react-hooks";
+import ProjectByNameWithDeployKeyQuery from "../../lib/query/ProjectByNameWithDeployKey";
+import { Footer, StyledNotificationWrapper } from '../Organizations/SharedStyles';
+import getConfig from "next/config";
+const { WEBHOOK_URL } = getConfig().publicRuntimeConfig;
+
+const DEPLOY_ENVIRONMENT_BRANCH_MUTATION = gql`
+  mutation (
+    $project: String!,
+    $branch: String!,
+    ) {
+    deployEnvironmentBranch(input: {
+      project:{
+        name: $project
+      }
+      branchName: $branch
+    }
+    )
+  }
+`;
+
+const customStyles = {
+  content: {
+    width: '50%',
+  },
+};
+
+const hashValue = (value) => {
+  let hashedVal = "●";
+  for (let l = 0; l < value.length; l++) {
+    hashedVal += "●";
+  }
+  return hashedVal;
+};
+
+const NewEnvironment = ({
+ inputProjectName,
+ productionEnvironment,
+ environmentCount,
+ inputBranchName,
+ setBranchName,
+ open,
+ openModal,
+ closeModal,
+ refresh,
+ setClear,
+ }) => {
+  const webhookURL = WEBHOOK_URL ? WEBHOOK_URL : 'https://webhook-handler-lagoon-master.lagoon.ch.amazee.io/';
```

Comment:
Should be generic just in case
```suggestion
  const webhookURL = WEBHOOK_URL ? WEBHOOK_URL : 'https://webhook-handler.example.com';
```